dev-environments/http-proxy-plain-http-upstream/Makefile

```diff
@@ -6,7 +6,7 @@ WORKDIR := $(patsubst %/,%,$(dir $(MKFILE_PATH)))
 DOCKER ?= $(shell which docker 2> /dev/null || echo "docker")
 
 gateway: ## run gateway configured to access upstream powered with TLS
-	$(DOCKER) compose -f docker-compose.yml run --service-ports gateway
+	$(DOCKER) compose -f docker-compose.yml up --attach gateway
```

Comment:
I  tend to keep unrelated changes to a separate commit or PR, so it is easier to check commit history. Just a reference of style, no big issue.

Reply:
No changes needed, let's fix the verification steps and update changelog and we can merge this PR

Reply:
You are right. Yes, changes are needed. 

https://github.com/3scale/APIcast/pull/1442